SharedCode/src/commonMain/kotlin/ApplicationPresenter.kt

```diff
@@ -85,4 +85,21 @@ class ApplicationPresenter : ApplicationContract.Presenter() {
 
         return jsonResponse
     }
+
+    fun onBuyButton(
+        outBound: String,
+        inBound: String,
```

Comment:
I think inbound and outbound are single words, so no need to capitalise Bound here.